bpf/bpf_lxc.c

```diff
@@ -690,6 +688,21 @@ static inline int __inline__ ipv4_policy(struct __sk_buff *skb, int ifindex, __u
 	if (ret < 0)
 		return ret;
 
+#ifdef LXC_NAT46
+	if (skb->cb[CB_NAT46_STATE] == NAT46) {
+		union v6addr dp = LXC_IP;
+		__u32 lxc_id = derive_lxc_id(&dp);
+
+		ret = ipv4_to_ipv6(skb, ip4, 14, &dp);
+		if (IS_ERR(ret))
+			return ret;
+
+		cilium_trace_capture(skb, DBG_CAPTURE_AFTER_V46, skb->ingress_ifindex);
+		tail_call(skb, &cilium_policy, lxc_id);
```

Comment:
Can we use `LXC_ID` directly here? `lxc_id` should always be the ID of the container we are forwarding into, right? I think you can save yourself the hassle of extracting the lxc id again.